lib/matplotlib/animation.py

```diff
@@ -935,6 +935,9 @@ def __init__(self, fig, event_source=None, blit=False):
         if self._blit:
             self._setup_blit()
 
+        # Boolean indicating if the animation is paused.
+        self._paused = False
```

Comment:
Is this actually needed? The individual `pause` and `resume` look like they would work fine for repeated calls. Just trying to limit how much state we're managing...

Reply:
Yea good point, I just figured it was better practice to add a bit of state to prevent weird behaviors from multiclicks. I wasn't sure if they're ok so I went with the more defensive option first. I removed it in the most recent commit.

Reply:
Totally reasonable. Just trying to go with minimum viable solution. 😉